### PR TITLE
Kf readme

### PR DESCRIPTION
**Pull request by TheKaterTot**
This needs approval from @meyerhoferc to ensure accuracy. It also includes an image that only shows up when the markdown is rendered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+
+You can use the live site at http://legislate.money
+
+**Setup**
+
+If you are new to Python/Django, you can use this guide to help set up your environment:
+
+    http://www.obeythetestinggoat.com/book/pre-requisite-installations.html
+
+Will Legislate For Money uses Python 3 and Django 1.11rc1.
```
Comment (meyerhoferc) [inline README.md]: Not sure how specific we want to be, but we are using Python 3.6

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+    shutil
+
+To set up the database and seed data:
+
+    python manage.py migrate
+    python manage.py seed_legislators
+
+**Usage**
+
+Note: To run the test suite, you will need to install vcrpy and selenium>3.
```
Comment (meyerhoferc) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -1,2 +1,57 @@
...
+In your virtual environment, install:
+
+    requests
+    
+    shutil
+
+To set up the database and seed data:
+
+    python manage.py migrate
+    python manage.py seed_legislators
```
Comment (meyerhoferc) [inline README.md]: Also need `python manage.py get_images`

other note: the images need to be added to the .gitignore. I'll try to include that in my next PR

**Comment by TheKaterTot** [comment]
Changes made! Thanks @meyerhoferc

**Pull request merged**